server/testutils/factories/cas1SpaceBooking.ts

```diff
@@ -0,0 +1,48 @@
+import { Factory } from 'fishery'
+import { faker } from '@faker-js/faker'
+import { Cas1KeyWorkerAllocation, Cas1SpaceBooking, Person } from '@approved-premises/api'
+import { fullPersonFactory } from './person'
+import cas1SpaceBookingDatesFactory from './cas1SpaceBookingDates'
+import cas1SpaceBookingRequirementsFactory from './spaceBookingRequirements'
+import userFactory from './user'
+
+import staffMemberFactory from './staffMember'
+import { DateFormats } from '../../utils/dateUtils'
+
+const addTime = (date: Date): Date => new Date(date.getTime() + faker.number.int({ max: 60 * 60 * 24 * 1000 }))
+
+export default Factory.define<Cas1SpaceBooking>(() => {
+  const startDateTime = addTime(faker.date.soon({ days: 90 }))
+  const endDateTime = addTime(faker.date.soon({ days: 365, refDate: startDateTime }))
```

Comment:
What's the purpose of using `addTime` here? I'm not sure I get it... If this is to push the dates into the future a bit, there is an `addDays` utility somewhere, I believe!

Reply:
The `actual` dates are actually date/times. This adds a random time of up to 24 hours to the dates.